### PR TITLE
x11-misc/sxcs: add v1.1.0

### PR DESCRIPTION
**Pull request by N-R-K**
- [x] I can submit this contribution in agreement with the [Copyright Policy](https://www.gentoo.org/glep/glep-0076.html#certificate-of-origin).
- [x] I have certified the above via adding a `Signed-off-by` line to *every* commit in the pull request.
- [x] This contribution has not been created with the assistance of Natural Language Processing artificial intelligence tools, in accordance with the [AI policy](https://wiki.gentoo.org/wiki/Project:Council/AI_policy).
- [x] I have run `pkgcheck scan --commits --net` to check for issues with my commits.

### CODE REVIEW

File: x11-misc/sxcs/sxcs-1.1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+RDEPEND="
+	x11-libs/libX11
+	x11-libs/libXcursor
+"
+DEPEND="${RDEPEND}"
+
+src_compile() {
+	edo $(tc-getCC) -o sxcs sxcs.c ${CFLAGS} ${LDFLAGS} \
+	    -l X11 -l Xcursor || die "Compilation failed"
```
Comment (Flowdalic) [inline x11-misc/sxcs/sxcs-1.1.0.ebuild:26]: No need to chain for here, edo will handle that for you

Comment (N-R-K) [reply x11-misc/sxcs/sxcs-1.1.0.ebuild]: Ah, right. Fixed.

File: x11-misc/sxcs/sxcs-1.1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# Copyright 2021-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit toolchain-funcs edo shell-completion
```
Comment (Flowdalic) [inline x11-misc/sxcs/sxcs-1.1.0.ebuild:6]: Sort inherits. Note that tools like `pkgcheck` should inform you about this.

Comment (N-R-K) [reply x11-misc/sxcs/sxcs-1.1.0.ebuild]: > Sort inherits.

Done.

> Note that tools like pkgcheck should inform you about this.

I ran `pkgcheck scan --commits HEAD` before opening PR, it didn't say anything about sorting inherits.

I *did* get a warning about "codeberg" being invalid for `remote-id` but it's been set to codeberg for a while now and I don't think it used to warn on it before, so I'm not sure what to do about it.

Comment (Flowdalic) [reply x11-misc/sxcs/sxcs-1.1.0.ebuild]: > I ran `pkgcheck scan --commits HEAD` before opening PR, it didn't say anything about sorting inherits.

I think it used to do that, but I couldn't get it to spit out a warning either. Strange.

Thanks for the fixes.



> I _did_ get a warning about "codeberg" being invalid for `remote-id` but it's been set to codeberg for a while now and I don't think it used to warn on it before, so I'm not sure what to do about it.

I don't get such a warning. And in fact, 'codeberg' appears in https://gitweb.gentoo.org/data/dtd.git/tree/metadata.dtd#n59

Maybe some tooling is outdated on your end?

Comment (N-R-K) [reply x11-misc/sxcs/sxcs-1.1.0.ebuild]: > Maybe some tooling is outdated on your end?

Likely. I don't do system updates very often so my local pkgcheck isn't up to date. I'll get it sorted out.

**Pull request closed without merging**